packages/angular/projects/clr-angular/src/forms/combobox/combobox-container.ts

```diff
@@ -38,7 +42,7 @@ import { IfControlStateService } from '../common/if-control-state/if-control-sta
         shape="check-circle"
         aria-hidden="true"
       ></clr-icon>
-      <ng-content select="clr-control-helper" *ngIf="showHelper"></ng-content>
+      <ng-content select="clr-control-helper" *ngIf="displayHelperInformation()"></ng-content>
       <ng-content select="clr-control-error" *ngIf="showInvalid"></ng-content>
       <ng-content select="clr-control-success" *ngIf="showValid"></ng-content>
     </div>
```

Comment:
Its not in the diff, but with the other changes you can remove the `controlSuccessComponent` content child property from all containers except abstract.